importer/importer/services/fhir_keycloak_api.py

```diff
@@ -99,7 +99,7 @@ def request(self, **kwargs):
         # TODO - spread headers into kwargs.
         headers = {"content-type": "application/json", "accept": "application/json"}
         response = self.api_service.oauth.request(headers=headers, **kwargs)
-        if response.status_code == 401 or '<html class="login-pf">' in response.text:
+        if response.status_code == 401 or 'login' in response.text:
```

Comment:
Initial line only worked for a html response, this should work even for a json response